app/assets/stylesheets/main.scss

```diff
@@ -32,8 +31,99 @@ body {
       font-size: 25px;
     }
   }
+
+  ul{
+    list-style:none;
+    display: inline-block;
+  }
+
+  ul li{
```

Comment:
Use SCSS scoping. If you have 3 rules, `ul`, `ul li`, and `ul li:hover`, you can just do:
```scss
ul {
  ...
  li {
    ...
    &:hover {
      ...
    }
  }
}
```